templates/_macros/vf_basic-section.jinja

```diff
@@ -330,7 +332,7 @@
   {%- set has_label = label_text|length > 0 -%}
   {%- set has_subtitle = subtitle_text|length > 0 -%}
 
-<section class="{{ padding_classes }}">
+<section class="{{ padding_classes }}"{% for attr, value in attrs.items() %} {{ attr }}="{{ value }}"{% endfor %}>
```

Comment:
As written, if a user specified a class attr there will be two competing class attrs on the element, instead of merging the pattern-defined and user-defined classes. For this reason, when I apply any attributes to the element in the pattern, I tend to also merge in the attribute from attrs if present, then skip adding it in the loop. Check the example I linked in my previous comment